### PR TITLE
eset like forums, businessinsider domains, wyborcza

### PR DESCRIPTION
**Pull request by FadeMind**
- https://ithardware.pl/
- https://forum.ithardware.pl/

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -4177,8 +4180,13 @@ CSS
...
 .ipsMenu_sep, 
 .ipsMenu_title, 
-.ipsPadding {
-	background-color: var(--darkreader-neutral-background) !important;
+.ipsPadding,
+.cUserHovercard {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+a[data-ipshover-target*="profile"] {
+    color: #fff !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4188]: 👀 `var(--darkreader-neutral-text)`

Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config]: thanks

**Comment by FadeMind** [comment]
https://www.trojmiasto.pl/

**Comment by FadeMind** [comment]
https://www.businessinsider.com/?IR=C
https://businessinsider.com.pl
https://businessinsider.co.za
https://businessinsider.es
https://businessinsider.jp
https://businessinsider.mx
https://insider.com
https://it.businessinsider.com






etc.

**Comment by FadeMind** [comment]
- https://wyborcza.biz/Gieldy/0,114514,,,,WIG.html
- https://wyborcza.biz/Waluty/0,111138,8932151,,,Kursy_srednie_walut_NBP,A.html#TRNavSST
- https://wyborcza.pl/0,0.html

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -1753,6 +1753,44 @@ img[alt="Burnaware"]
 
 ================================
 
+businessinsider.com
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:1756]: Yeez so many sites, just trow an 

```
businessinsider.*
```

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -4177,8 +4218,12 @@ CSS
...
 .ipsMenu_sep, 
 .ipsMenu_title, 
-.ipsPadding {
-	background-color: var(--darkreader-neutral-background) !important;
+.ipsPadding,
+.cUserHovercard {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+a[data-ipshover-target*="profile"] {
+    var(--darkreader-neutral-text) !important;
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:4226]: this ain't working.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config]: done 😁

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (FadeMind) [reply src/config/dynamic-theme-fixes.config]: `a[data-mentionid]`

**Review by Gusted — CHANGES_REQUESTED** [review]
A lot of this images are already inverted by default. I'm not sure how this isn't the case for you as this patch breaks more than it fixes.

**Comment by FadeMind** [comment]
@Gusted business insider without fixes use black instead white. I can move them to another separated pr if you like. 
I used selectors only needed to invert. Nothing else. (logos, icons). 
Your call.

**Comment by FadeMind** [comment]
Default - no changes applied
https://www.businessinsider.co.za/apple-imac-and-ipad-production-will-be-delayed-chip-shortage-2021-4



https://www.businessinsider.com/biden-first-100-days-how-compare-trump-obama-bush-clinton-2021-4?IR=T#while-trump-was-a-prolific-golfer-during-his-presidency-biden-made-his-first-trip-to-a-golf-course-nearly-three-months-into-his-first-term-9




https://www.businessinsider.jp/

**Comment by Gusted** [comment]
Hey! That's weird, we might switch back to remove the `bussinessinsider.*` because on .com and e.g. .nl it works without the fixes.

**Comment by FadeMind** [comment]
@Gusted  thats why  I specify domains ;)

**Review by Gusted — APPROVED** [review]

**Comment by FadeMind** [comment]
@Gusted I will pull one more fix for https://www.businessinsider.com.au/ missed in this PR.

**Pull request merged**